api/build.gradle

```diff
@@ -367,7 +367,7 @@ dependencies {
     __swaggerCodegenV2__ "io.swagger:swagger-codegen-cli:$project.ext.SWAGGER_2_CODEGEN_VERSION"
     __swaggerCodegenV3__ "io.swagger.codegen.v3:swagger-codegen-cli:$project.ext.SWAGGER_3_CODEGEN_VERSION"
 
-    compile 'ch.qos.logback:logback-classic:1.4.3'
+    compile 'ch.qos.logback:logback-classic:1.2.11'
```

Comment:
I also tried a smaller downgrade to 1.3.3 but that didn't work.  I didn't investigate why.